docs/root/intro/arch_overview/ext_authz_filter.rst

```diff
@@ -0,0 +1,41 @@
+.. _arch_overview_ext_authz:
+
+External Authorization
+======================
+
+* :ref:`Network filter v2 API reference <envoy_api_msg_config.filter.network.ext_authz.v2.ExtAuthz>`
```

Comment:
Instead of looking into the proto docs here, can you link into the generic filter configuration docs (which then link to proto docs)?

Reply:
good point. i've made the change.